index.rst

```diff
@@ -288,6 +288,13 @@ Welcome to PyTorch Tutorials
    :link: advanced/static_quantization_tutorial.html
    :tags: Image/Video,Quantization,Model-Optimization
 
+.. customcarditem::
```

Comment:
If this will be on the recipes page, we'll need to move this over to recipes_index.rst instead of index.rst which is for the longer end-to-end tutorials: https://github.com/pytorch/tutorials/blob/master/recipes_source/recipes_index.rst